resources/public/cljdoc.css

```diff
@@ -118,3 +118,16 @@
   bottom: 0;
   left: -2rem;
 }
+
+.raw {
+    font-family: "SFMono-Regular", Consolas, "Liberation Mono", Menlo, Courier, monospace;
+    padding: 1rem;
+    background-color: rgb(246, 248, 250);
+    border-radius: 3px;
+    font-size: 0.875rem;
+    overflow-x: scroll;
+}
+
+.hide {
+    display: none;
+}
```

Comment:
cljdoc mostly uses Tachyons (see [ADR #0011](https://github.com/cljdoc/cljdoc/blob/master/doc/adr/0011-use-tachyons-as-css-foundation.md)) for styling, basically using utility classes right in the markup. I think in this case we probably want something like the following:

- `.code`([tachyons docs](http://tachyons.io/docs/typography/font-family/)) for monospaced font, which is also whats used for arglists. Although an argument could be made that your stack is nicer since it uses SFMono when available. 
- `.pa3` 1rem padding
- `.br2` for 4px border-radius
- `.f6` for font-size
- `.overflow-x-scroll` for, well... overflow-x: scroll

For the background color you can see if you find something here: https://tachyons.io/docs/themes/skins/

finally you can eliminate the `.hide` class by using the `display` helpers: 

- `.dn` display none
- `.db` display block

Reply:
I updated the styling to use tachyons